internal/vendor/wiz.go

```diff
@@ -203,15 +203,15 @@ type SAMLGroupMappingUpdateInput struct {
 // CreateSAMLIdentityProviderInput struct -- updates
 type CreateSAMLIdentityProviderInput struct {
```

Comment:
The golang struct should mirror the GraphQL schema when deciding whether to omit empty attributes.  E.g., allowManualRoleOverride is optional, so the golang struct should have the omitempty tag.

```
input CreateSAMLIdentityProviderInput {
  name: String!
  issuerURL: String
  loginURL: String!
  logoutURL: String
  useProviderManagedRoles: Boolean!
  allowManualRoleOverride: Boolean
  certificate: String!
  domains: [String!]
  groupMapping: [SAMLGroupMappingCreateInput!]
  mergeGroupsMappingByRole: Boolean
}
```